### DISCUSSION TITLE
Things to leave outside the app

### DISCUSSION THREAD

**Discussion by luisivan** [nation3/court#17] [Specs]
Just wanted to write down a couple things that I think should happen outside of the app:
- Submitting evidence: Parties can submit evidence to jurors off-chain. We can have an up-to-date wiki page with information about the current jurors, including their PGP keys and the email where encrypted evidence can be submitted for them to access.
- Appealing a ruling by the jurors: $veNATION holders, via the Nation3 DAO, can appeal a ruling done by the jurors. That means submitting a vote to the DAO, which can be done either using [Frame](https://frame.sh)'s integration with Aragon, or using [Aragon Console](https://client.aragon.org/#/nation3/console). For now, I think keeping this as a somewhat technical task makes sense, as it's a mechanism that should only be used in force majeaure situations.

The goal is to reduce complexity so that we can ship a v1, test the hypothesis, and take it from there. Thoughts @xPi2 @gregfromstl?

**Comment by xPi2** [comment]
I agree with the first point but I have doubts with the second one.
I want to implement a simple appeal system in v1. It shouldn't require extra complexity. An appeal will simply prevent the resolution committed by the Court for happening and require instead the input you described.
So yes, the actual resolution from the DAO it's outside the App but not the appeal (IMO).
